### ISSUE TITLE
Double break line when writing error logs

### ISSUE THREAD

**Issue by jptosso** [corazawaf/coraza-caddy#87]
Errors are printed with double break line using console log format. 
```
2023/07/14 12:49:16.929 ERROR   http.handlers.waf       [client "xxxxx"] Coraza: Access denied (phase 1).  [file ""] [line "8164"] [id "123"] [rev ""] [msg ""] [data ""] [severity "emergency"] [ver ""] [maturity "0"] [accuracy "0"] [hostname ""] [uri "/test5"] [unique_id "lplsWtaxdPAZBEMV"]

2023/07/14 12:54:49.311 ERROR   http.handlers.waf       [client "xxxx"] Coraza: Access denied (phase 1).  [file ""] [line "8164"] [id "123"] [rev ""] [msg ""] [data ""] [severity "emergency"] [ver ""] [maturity "0"] [accuracy "0"] [hostname ""] [uri "/test5"] [unique_id "ipPVdICWZdSKCKnq"]

2023/07/14 12:56:30.860 ERROR   http.handlers.waf       [client "xxxx"] Coraza: Access denied (phase 1).  [file ""] [line "8164"] [id "123"] [rev ""] [msg ""] [data ""] [severity "emergency"] [ver ""] [maturity "0"] [accuracy "0"] [hostname ""] [uri "/test5"] [unique_id "HUDskOlODMNnKEqb"]
```

**Comment by jcchavezs** [comment]
Awesome!

**Comment by M4tteoP** [comment]
The upstream fix is part of Coraza 3.0.4 and Caddy has been just updated via https://github.com/corazawaf/coraza-caddy/pull/102. Closing as completed!
